java-persistence/src/main/java/com/whitecloak/training/inventory/controller/response/ItemResource.java

```diff
@@ -1,10 +1,24 @@
 package com.whitecloak.training.inventory.controller.response;
 
+import com.whitecloak.training.inventory.persistence.entity.CategoryEntity;
+import com.whitecloak.training.inventory.persistence.entity.UserEntity;
+
 public class ItemResource {
 
     private long id;
-    private String name;
-    private long ownerId;
+    private String itemName;
+    private UserEntity owner;
+    private CategoryEntity category;
+
+    public ItemResource() {
+    }
+
+    public ItemResource(long id, String itemName, UserEntity owner, CategoryEntity category) {
```

Comment:
Entity has leaked to the web layer.

You should compose a resource with resource - Replacing `UserEntity` with `UserResource` will do. (also in the category)